go/vt/vtorc/logic/keyspace_shard_discovery.go

```diff
@@ -22,35 +22,13 @@ import (
 
 	"golang.org/x/exp/maps"
 
-	"vitess.io/vitess/go/stats"
 	"vitess.io/vitess/go/vt/key"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/topo"
 	"vitess.io/vitess/go/vt/topo/topoproto"
 	"vitess.io/vitess/go/vt/vtorc/inst"
 )
 
-var statsShardsWatched = stats.NewGaugesFuncWithMultiLabels("ShardsWatched",
```

Comment:
This stat was added by me very recently and no release has occurred since, so there's no deprecation plan